dev/benchmarks/complex_layout/lib/main.dart

```diff
@@ -611,7 +611,7 @@ class GalleryDrawer extends StatelessWidget {
           new DrawerItem(
             icon: new Icon(Icons.hourglass_empty),
             selected: timeDilation != 1.0,
-            onPressed: () { ComplexLayoutApp.of(context).toggleAnimationSpeed(); },
+            onPressed: ComplexLayoutApp.of(context).toggleAnimationSpeed,
```

Comment:
This is not equivalent. The original code does the lookup at the time of the tap. The new code does the lookup at build time. The lookup is expensive. The lookup can also change result based on changes outside this function.